ceph-releases/ALL/centos/daemon/__DOCKERFILE_PREINSTALL__

```diff
@@ -1 +0,0 @@
-COPY --from=builder /go/bin/forego /usr/local/bin/forego
```

Comment:
Does this file need to exist but should be empty?

Reply:
yes, it is included in the base dockerfile

see https://github.com/ceph/ceph-container/blob/master/src/daemon/Dockerfile#L16